packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/overview/index.md

```diff
@@ -16,6 +16,6 @@ After you have prepared and tested your SCIM integration, and you want to make i
 
 ## Support
 
-If you need additional assistance, read through our [SCIM FAQ](/docs/concepts/scim/faqs/) page, post your question on the [Okta Developer Forums](https://devforum.okta.com/), or email us at <developers@okta.com>.
+If you need additional assistance, read through our [SCIM FAQ](/docs/concepts/scim/faqs/) page or post your question on the [Okta Developer Forums](https://devforum.okta.com/).
```

Comment:
Do we refer to plural forums or just one forum? The previous entry was singular `Okta Developer Forum` and this one is plural `Okta Developer Forums`.

Reply:
We have only one forum, so it should be singular

Reply:
Thanks for the feedback. I'm replacing all plural references with singular.